configure.ac

```diff
@@ -152,9 +152,15 @@ AC_ARG_ENABLE([lastlog],
 )
 
 AC_ARG_ENABLE([logind],
-	[AS_HELP_STRING([--enable-logind],
-		[enable logind @<:@default=yes@:>@])],
-	[enable_logind="${enableval}"],
+	[AS_HELP_STRING([--disable-logind],
+		[disable logind integration])],
+	[
+		AS_CASE([${enableval}],
+			[yes],[],
+			[no],[],
+			[AC_MSG_ERROR([bad parameter value for --enable-logind=${enableval}])]
```

Comment:
We say there is a bad value but log message will not say which values are allowed :-)
```
checking location of faillog/lastlog/wtmp... /var/log
configure: error: bad parameter value for --enable-logind=yesd
```

Reply:
User can guess that `./configure --help` can suggest something.
Really, do we need to explain that "no value, or no parameter, or --disable-logind, or --enable-logind=yes, or --enable-logind=no should be used"?

Reply:
Sorry, I asked you to slightly improve error message.

Feel free to resolve the comment/discussion.

Feature wise PR is OK

LGTM

Reply:
@Karlson2k 

Up to you.  Once you reject the suggestion, or improve the error message, let me know and I'll merge.

Thank you both!!

Reply:
I would have accepted any of them.  I'm not sure which I prefer.  :)